core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -387,7 +386,15 @@ public OutputStream openLogFile() {
 
     private final Object channelLock = new Object();
 
-    public void setChannel(InputStream in, OutputStream out, TaskListener taskListener, Channel.Listener listener) throws IOException, InterruptedException {
+    /**
+     * Creates a {@link Channel} from the given stream and sets that to this agent.
+     *
+     * Same as {@link #setChannel(InputStream, OutputStream, OutputStream, Channel.Listener)}, but for
```

Comment:
I think you have the comment the wrong way around?

`Same as {@link #setChannel(InputStream, OutputStream, OutputStream, TaskListener)} but for {@link Channel.Listener}`